examples/Huggingface_Transformers/Image_classification_docker/HF-only/README.md

```diff
@@ -0,0 +1,158 @@
+cd HF-models/vitxxsmall/
+git lfs install
+git lfs pull
+cd ../..
+```
+
+⚠️ **IMPORTANT NOTE** ⚠️: The folder in which the 🤗 model repo is cloned & the name of the `.mar` file should be EXACTLY the same, this constraint was necessary to register new models with `curl POST` requests flexibly (i.e, during server intialization as well as afterwards).
+
+
+Create a Torchserve model archive with the model handler file (`scripts/torchserve_vitxxsmall_handler.py` in our example) along with relevant dependencies in `requirements.txt` (like 🤗 transformers).  
+
+**NOTE:** Since we are not giving a pretrained checkpoint as a `.pth` file (as it would be downloaded from 🤗 in the `initialize` method of our `torchserve_vitxxsmall_handler.py`), the `--serialized-file` option is redundant as we do not use the context in our handler. 
+```
+touch dummy_file.pth
+torch-model-archiver --model-name vitxxsmall --serialized-file dummy_file.pth --version 1.0 --handler scripts/torchserve_vitxxsmall_handler.py --export-path model-store -r requirements.txt
+rm -f dummy_file.pth
+```
+
+Since the `load_models` attribute of `config.properties` (that was passed to the `docker run` command while starting the Torchserve server) is set to "standalone", the Torchserve server is initialized without any models initially (even though `model-store` might contain `.mar` files). 
+
+Registering the MobileViT XX Small model on the Torchserve server (more details [here](https://github.com/pytorch/serve/blob/master/docs/management_api.md#register-a-model)) with `max_batch_delay` in milliseconds, which is the time the Torchserve server waits to bundle concurrent inference requests into a batch with maximum size of `batch_size` (i.e, `preprocess` of the handler always receives list of requests with length <= `batch_size`):
+```
+curl -X POST "localhost:8081/models?url=vitxxsmall.mar&batch_size=8&max_batch_delay=10&initial_workers=1"
+```
+
+In case of bugs, recently created container can be accessed to check the logs for debugging, metrics etc(check the [logging documentation](https://github.com/pytorch/serve/blob/master/docs/logging.md) for details).  
+In case `torch-model-archiver` is unavailable in the AMI (or any cloud instance), the Torchserve container can be accessed to create the model archive in the `model-store`, which is mounted as shared memory between the AMI/VM and container (might need modifications to [`mount`](https://docs.docker.com/storage/bind-mounts/#choose-the--v-or---mount-flag) flag )
+
+
+```
+serve_cont_id=$(docker ps -l -q) 
+docker exec -it $serve_cont_id /bin/bash
+cat logs/model_log.log
+```
+
+## Managing registered models
+
+Run the following commands to check the registered models :
+```
+curl http://127.0.0.1:8081/models/
+
+curl http://127.0.0.1:8081/models/vitxxsmall/
+```
+
+The models running on the Torchserver server can be managed with a [gRPC API](https://github.com/pytorch/serve/blob/master/docs/management_api.md#scale-workers).
```

Comment:
nit: project is called TorchServe

Reply:
A typo that I must have missed, would fix asap.